ping_identity/README.md

```diff
@@ -0,0 +1,52 @@
+# Ping Identity - Datadog API Connector
+
+## Overview
+
+Ping Identity delivers intelligent identity solutions for the enterprise. We enable companies to achieve Zero Trust identity-defined security and more personalized, streamlined user experiences.
+
+Datadog integrates directly with the PingOne DaVinci platform through the Datadog API connector. PingOne DaVinci is an orchestration platform that lets you create flows using connections and logical operators. These flows guide users through defined processes that can present customized pages, modify values, or perform other actions. 
+
+The Datadog API connector boasts over 350+ unique capabilites, each with a unique Datadag API endpoint to complete CRUD actions for many different Datadog components.
+
+## Setup
+
+### Installation
+
+Installation with PingOne's DaVinci platform is simple and easy with Ping's drag and drop functionality on a flow canvas. Follow the below steps to configure Datadog within DaVinci:
+
+1. On a flow, select the black "+" button in the bottom left corner of the flow canvas. 
+
+2. Search for Datadog and select it to add it to the canvas.
```

Comment:
```suggestion
2. Search for "Datadog" and select it to add it to the canvas.
```
Can you replace "it" with a word that describes how "Datadog" appears? Is it a list item, a button, a tile, or something else?